frontend/cypress/integration/featureFiles/istio_config.feature

```diff
@@ -13,6 +13,7 @@ Feature: Kiali Istio Config page
     And user selects the "bookinfo" namespace
 
   @bookinfo-app
+  @selected
```

Comment:
@nrfox Did we mean to leave in this and a few other `@selected`  annotations?

Reply:
Whoops